docs/source/features/fp8.rst

```diff
@@ -0,0 +1,40 @@
+
+.. list-table:: FP8 arguments
+   :widths: 25 25 50
+   :header-rows: 1
+
+   * - Argument
+     - Description
+   * - transformer_engine
+     - Boolean argument that needs to be set in order to enable TE and related functionality. All following arguments are ignored if this is not set to True.
+   * - fp8
+     - Enables FP8 training. For transformer networks, the QKV, projection, FC1, and FC2 matrix multiplications are executed using the 4th generation H100 tensor cores with FP8 support.
+   * - fp8_e4m3
+     - Training recipe format for FP8. Activations, weights, and gradient tensors use the E4M3 format.
+   * - fp8_hybrid
+     - Training recipe format for FP8. Activations and weight tensors use the E4M3 format, whereas gradient use the E5M2 format to satisfy the additional dynamic range requirement for backward tensors. This is the default setting.
+   * - fp8_margin
+     - The scaling factor for FP8 tensors can be shifted by a factor of $2 ^ {margin}$ using this argument.
+   * - fp8_amax_history_len
+     - Window size for the amax history. This many instances of most recent amaxes (absolute max) are stored per tensor.
+   * - fp8_amax_compute_algo
+     - "max" or "most_recent". Specifies how to choose an amax from the given history.
+   * - reduce_amax
+     - Whether or not to perform an allreduce on the amax (absolute max) values for the FP8 tensors. Since the amax is directly used to compute the scaling factor for FP8 tensors, setting this ensures that the scaling factors for a tensor remain synchronized across devices in multi-GPU training configurations.
+   * - fp8_params
+     - Whether or not to store module level parameters in FP8. Setting this may result in a lower memory consumption since this eliminates the need to store a copy of weights in higher precision (> half) for cases in which these are maintained externally anyway, such as master parameters in the optimizer. For details, refer to the `fp8_model_init <https://docs.nvidia.com/deeplearning/transformer-engine/user-guide/api/pytorch.html#transformer_engine.pytorch.fp8_model_init>`_ API in TE.
+
+Resources
+^^^^^^^^^
+
+- `TE documentation <https://docs.nvidia.com/deeplearning/transformer-engine/user-guide/index.html>`_.
```

Comment:
Change "- `TE documentation <https://docs.nvidia.com/deeplearning/transformer-engine/user-guide/index.html>`_.: to :- `TE documentation <https://docs.nvidia.com/deeplearning/transformer-engine/user-guide/index.html>`_"

Remove the period at the end of the sentence